app/code/community/Reverb/ReverbSync/Helper/Orders/Creation/Address.php

```diff
@@ -22,8 +26,10 @@ public function addOrderAddressAsShippingAndBillingToQuote(stdClass $reverbOrder
         $shippingAddressObject = $reverbOrderObject->shipping_address;
         if (!is_object($shippingAddressObject))
         {
-            $error_message = $this->__(self::ERROR_NO_ADDRESS);
-            throw new Exception($error_message);
+            // In this event, we still want to create the Magento order. We will populate the address fields with a
+            //      fake placeholder value
+            $this->_addNoAddressToOrderObject($reverbOrderObject);
+            $shippingAddressObject = $reverbOrderObject->shipping_address;
```

Comment:
Well what you're doing here is checking whether we have a shipping address. I think there is a small chance that we do have most of the shipping address but it's missing a country code. We're trying to fix that on our side but it's possible for legacy orders. Can we make a thing that defaults any field that's missing to some value like "Unknown". For country code if it's missing you can use "XX"